base_ubl/tests/test_ubl_generate.py

```diff
@@ -29,3 +29,83 @@ def test_ubl_generate(self):
         self.env["base.ubl"]._ubl_add_customer_party(
             invoice.partner_id, False, "AccountingCustomerParty", xml_root, ns
         )
+
+    def create_test_invoice(
```

Comment:
Please credit Alexis for this code as it was originally added to account_tax_unece in https://github.com/OCA/community-data-files/pull/24

Reply:
We'd like to get this merged. We'll create a superseding PR with this change.

Reply:
@phuctranerp created superseeding PR: https://github.com/OCA/edi/pull/361